frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js

```diff
@@ -184,10 +184,17 @@ export class ViewPartyRelationships extends Component {
     });
   };
 
+  // Since end date is stored at yyyy-mm-dd, comparing current Date() to
+  // the the start of the next day ensures appointments ending today are displayed.
+  getTomorrow = (end_date) => {
```

Comment:
Small naming thing, but this is really more of a `getNextDay` since it accepts a date and gives the day after that.

Reply:
good point